client/src/Movies/MovieList.js

```diff
@@ -43,7 +54,7 @@ function MovieDetails({ movie }) {
         Metascore: <strong>{metascore}</strong>
       </div>
       <h3>Actors</h3>
-
+      
       {stars.map(star => (
         <div key={star} className="movie-star">
           {star}
```

Comment:
Try not to use names as map keys (unless you have a way of guaranteeing their uniqueness) - while unlikely, there is still the slim chance of two people having the exact same names, and you'll want to account for this possibility especially when drawing from a set of data that you cannot check for duplicates.